target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -110,6 +111,11 @@ mediatek_setup_macs()
 		wan_mac=$label_mac
 		lan_mac=$(macaddr_add "$label_mac" 2)
 		;;
+	zbtlink,zbt-z8103ax)
```

Comment:
You are already setting up MAC addresses in device tree, no need to do it again in userspace. Remove it here.